variables.tf

```diff
@@ -666,7 +666,9 @@ variable "runner_worker_docker_machine_instance" {
     start_script = Cloud-init user data that will be passed to the Runner Worker. Should not be base64 encrypted.
     subnet_ids = The list of subnet IDs to use for the Runner Worker when the fleet mode is enabled.
     types = The type of instance to use for the Runner Worker. In case of fleet mode, multiple instance types are supported.
-    volume_type = The type of volume to use for the Runner Worker.
+    volume_type = The type of volume to use for the Runner Worker. `gp2`, `gp3`, `io1` or `io2` are supported.
```

Comment:
suggestion: This drops `sc1` and `st1` support. I guess nobody uses the hard drives and the cloud default is `gp2`. So I am fine with that. But we should add a validation.

```hcl
validation {
  condition = contains(["gp2", "gp3", "io1", "io2"], var.runner_worker_docker_machine_instance.volume_type)
  message = "Supported volume types: gp2, gp3, io1 and io2"
}
```

Reply:
Great feedback, thank you. I have added the validation block.